src/Acuminator/Acuminator.Analyzers/StaticAnalysis/PXCodeFixProvider.cs

```diff
@@ -69,7 +70,9 @@ private async Task<Document> AddSuppressionComment(CodeFixContext context, Diagn
 
 
 			SyntaxTriviaList commentNode = SyntaxFactory.TriviaList(
-				SyntaxFactory.SyntaxTrivia(SyntaxKind.SingleLineCommentTrivia, string.Format(_comment, diagnostic.Id, "Description")),
+				SyntaxFactory.SyntaxTrivia(SyntaxKind.SingleLineCommentTrivia, 
+													string.Format(_comment, diagnostic.Id, 
+														_fixableDiagnosticIds.FirstOrDefault( x => x.Key == diagnostic.Id).Value.ToString())),
```

Comment:
@saloponov  the dictionary will be better here?

Reply:
Then you can use a lookup or an ImmutableDictionary<string, List<string>> (I wouldn't mind a simple dictionay for a private member) By the way the ImmutableList is not frequently used and usually ImmutableAttay is recommended

Reply:
I am using `ImmutableDictionary<string, string>` and function `dictionary.TryAdd(key, value)` now. It is no differences between diagnostic's names. I will change this collection if @k-s-popova changes resource file.